cmd/klotho/main.go

```diff
@@ -21,9 +21,10 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
-var root = &cobra.Command{
-	Use:  "klotho [path to source]",
-	RunE: run,
+type KlothoMain struct {
```

Comment:
This being in `package main` means it wouldn't be reusable, so maybe move this into `package cli` package?